README.md

```diff
@@ -1,14 +1,16 @@
-# Postgres
+# Deis Postgres v2
 
-A PostgreSQL database for use in the [Deis](http://deis.io) open source PaaS.
+Deis (pronounced DAY-iss) is an open source PaaS that makes it easy to deploy and manage applications on your own servers. Deis builds on [Kubernetes](http://kubernetes.io/) to provide a lightweight, easy and secure way to deploy your code to production.
 
-This Docker image is based on the official
-[postgres](https://registry.hub.docker.com/_/postgres/) image.
+For more information about the Deis workflow, please visit the main project page at https://github.com/deis/workflow.
 
-Master/slave replication and leader election code is largely inspired by [`compose/governor`](https://github.com/compose/governor).
+## Beta Status
 
-Please add any [issues](https://github.com/deis/postgres/issues) you find with this software.
+This Deis component is currently in beta status, and we welcome your input! If you have feedback, please submit an [issue][issues]. If you'd like to participate in development, please read the "Development" section below and submit a [pull request][prs].
 
+# About
+
+This component is a PostgreSQL database for use in Kubernetes. It builds on the official [postgres](https://registry.hub.docker.com/_/postgres/) Docker image. While it's intended for use inside of the Deis open source [PaaS](https://en.wikipedia.org/wiki/Platform_as_a_service), it's flexible enough to be used as a standlone pod on any Kubernetes cluster.
```

Comment:
can we change the last sentence to

> While it's intended for use inside of the Deis open source [PaaS](https://en.wikipedia.org/wiki/Platform_as_a_service), it's flexible enough to be used as a standalone docker container.

Since this is proven to be true thanks to the integration tests :)